lib/lorekeeper/fast_logger.rb

```diff
@@ -82,8 +83,6 @@ def write(message)
       @iodevice.write(message)
     end
 
-    private
```

Comment:
is removing this correct?

Reply:
@JordiPolo 
about this, AFAIU the `private` keyword doesn't work for inner classes, so that's why rubocop is removing it

The only official docs solution that I found was to use `private_constant` (it's in JP but I used Google translate  👀 )
https://docs.ruby-lang.org/ja/latest/method/Module/i/private_constant.html

`private_const` method is also supported in ruby `2.7` which is the minimum required one for lorekeeper

https://docs.ruby-lang.org/en/2.7.0/Module.html#method-i-private_constant

Reply:
they were not private before! 😎

Reply:
indeed!! 👀 

AFAIU from some [StackOverflow discussion](https://stackoverflow.com/questions/39716799/how-to-implement-private-inner-class-in-ruby), 

>Also, be warned that even with private_constant, the privateness means very little. Basically, the only thing it does is hide the constant from lists (LinkedList.constants) and direct resolution (LinkedList::Node). If one knows the name, they will be able to access it.

even adding `private_constant` doesn't make them 100% private
I am not a fan of depending on StackOverflow as a source of truth, but I hope the change makes sense 👀

Reply:
My life is a lie

Reply:
LOL 🤣